spt/OrangeBox/patterns.hpp

```diff
@@ -268,6 +268,7 @@ namespace patterns
 		    "51 56 6A 14 8B F1 E8 ?? ?? ?? ?? 8B 54 24 10 8B C8 0F B7 02 89 44 24 10 83 C4 04 89 4C 24 04 DB 44 24 0C",
 		    "5377866",
 		    "55 8B EC 51 56 57 6A 14 89 4D FC E8 ?? ?? ?? ?? 8B 7D 08 8B F0");
+		PATTERNS(CViewEffects__Shake, "4104-5135", "56 8B 74 24 ?? 8B 06 85 C0");
```

Comment:
Could you make the pattern a little longer pls? With the number of DLLs SPT has to run on I think it's best to be a bit more cautious about collisions.

Reply:
Done.

Reply:
Why did the `??` disappear?

Reply:
Well, I just copied the pattern and it works stably, I can back `??` if needed. 

![0](https://user-images.githubusercontent.com/58108407/111784279-af465b00-88d4-11eb-8840-47fe42608067.PNG)